src/dispatch/static/dispatch/src/incident/ParticipantSelect.vue

```diff
@@ -95,17 +96,20 @@ export default {
   },
 
   methods: {
-    loadMore() {
-      this.numItems = this.numItems + 5
-      this.fetchData()
+    customFilter(item, queryText) {
+      const name = item.individual.name.toLowerCase()
+      const email = item.individual.email.toLowerCase()
+      const searchText = queryText.toLowerCase()
+
+      return name.indexOf(searchText) > -1 || email.indexOf(searchText) > -1
     },
     fetchData() {
       this.loading = "error"
       let filterOptions = {
         q: this.search,
         sortBy: ["name"],
         descending: [false],
-        itemsPerPage: this.numItems,
+        itemsPerPage: -1,
```

Comment:
This is worth noting. We should check how many users are in production. It'll be faster if we can just cache them all in the browser so that's why I did this.

Reply:
Unless we have some sort of global cache, e.g. in a vuex store. I think this might end up generating quite a bit of data downloading.